### PR TITLE
Make process-based steel default; update macro parameters

### PR DESCRIPTION
**Pull request by JakobBD**
## Purpose of this PR

- Make process-based steel (`cm_subsec_model = processes`) the default option.
- Change calibration file naming convention such that the default file name now refers to process-based steel (ces-based steel gets a file name extension) 
- New macro-economic parameters by @mleimbach 
- Minor developments on the process-based steel model (carbon capture tech now captures the co2 emitted from its own heating gas consumption) 

## Type of change

(Make sure to delete from the Type-of-change list the items not relevant to your PR)

- [ ] Bug fix 
- [ ] Refactoring
- [x] New feature 
- [ ] Minor change (default scenarios show only small differences)
- [x] Fundamental change
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the [coding etiquette](https://github.com/remindmodel/remind/blob/develop/main.gms#L80)
- [x] I performed a self-review of my own code
- [x] I explained my changes within the PR, particularly in hard-to-understand areas
- [x] I checked that the [in-code documentation](https://github.com/remindmodel/remind/blob/develop/main.gms#L120) is up-to-date
- [x] I adjusted the reporting in [`remind2`](https://github.com/pik-piam/remind2) where it was needed
- [x] I adjusted `forbiddenColumnNames` in [readCheckScenarioConfig.R](https://github.com/remindmodel/remind/blob/develop/scripts/start/readCheckScenarioConfig.R) in case the PR leads to deprecated switches
- [x] All automated model tests pass (`FAIL 0` in the output of `make test`)
- [x] The changelog `CHANGELOG.md` [has been updated correctly](https://gitlab.pik-potsdam.de/rse/rsewiki/-/wikis/Standards-for-Writing-a-Changelog)

## Further information (optional):

* Test runs are here: 

`/p/tmp/jakobdu/remind_pbs/output/`

* Comparison of results (what changes by this PR?):

### CODE REVIEW

File: scripts/start/calculate_CES_configuration.R [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # |  REMIND License Exception, version 1.0 (see LICENSE file).
 # |  Contact: remind@pik-potsdam.de
 calculate_CES_configuration <- function(cfg, path = getwd(), check = FALSE) {
-    CESstring <- paste0("indu_", cfg$gms$industry,ifelse(cfg$gms$cm_subsec_model_steel=="processes","pbs",""),"-",
+    CESstring <- paste0("indu_", cfg$gms$industry,ifelse(cfg$gms$cm_subsec_model_steel=="ces","CesSteel",""),"-",  # remove CesSteel suffix after process-based has been default for some months
```
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [inline scripts/start/calculate_CES_configuration.R:8]: You are redefining the meaning of the `CESstring`.  This is going to bite some project team when they have the least time to deal with it …

**Review by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q — COMMENTED** [review]
I think it would be generally beneficial to separate out different changes into different pull requests, instead of lumping them together.  In this case:
- setting new prtp/ies parameters
- removing explicit SE tracking for industry FE
- making PBS the default
From the PR title, nobody will guess all this is going on here.
Also, all three of these ought to be mentioned in the changelog.

**Comment by LaviniaBaumstark** [comment]
> I think it would be generally beneficial to separate out different changes into different pull requests, instead of lumping them together. In this case:
> 
>     * setting new prtp/ies parameters
> 
>     * removing explicit SE tracking for industry FE
> 
>     * making PBS the default
> 
> 
> From the PR title, nobody will guess all this is going on here. Also, all three of these ought to be mentioned in the changelog.

In general, I agree that the best way would be to have only one topic in each PR. But here `removing explicit SE tracking for industry FE`and `making PBS the default`are to closely related that they should be merged in one PR. In addition, `setting new prtp/ies parameters`would also require a recalibration merging it to one PR reduces the number of recalibrations and avoids having REMIND with no correlated CES parameters available.

File: scripts/start/calculate_CES_configuration.R (hunk 1 above)
Comment (LaviniaBaumstark) [reply scripts/start/calculate_CES_configuration.R:8]: the change of teh meaning is intended as pds will be our default from now on

**Comment by JakobBD** [comment]
I agree that Marian's changes should have been a different PR. 

> removing explicit SE tracking for industry FE

was a different PR #1659

Changelog will still be updated, that's why the box is still un-ticked. Fair point though, it wasn't clear from the PR that this is still planned.

**Comment by JakobBD** [comment]
# Compare Scenarios: 

Full PDFs are in cloud folder, link was posted in mattermost REMIND channel.

## NPI

image


## Pk Budget 650 

image
image

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
> > removing explicit SE tracking for industry FE
> 
> was a different PR #1659

Indeed it was.  I should look more closely at changed equations.

**Comment by JakobBD** [comment]
While I still believe some vetting of AMT CS2 results prior to merge would be nice, the PR is now ready for merge, as far as I can see. Calibration files in the public repo are already updated, and the new hash has been added to this PR.

**Review by LaviniaBaumstark — APPROVED** [review]

**Pull request merged**